docs/api/python/text/text.md

```diff
@@ -137,14 +137,14 @@ candidate indices may be obtained via
 .. autosummary::
     :nosignatures:
 
-    Glossary
+    CompositeEmbedding
 ```
 
 To get all the valid names for pre-trained embeddings and files, we can use
-[`TokenEmbedding.get_embedding_and_pretrained_file_names`](#mxnet.contrib.text.embedding.TokenEmbedding.get_embedding_and_pretrained_file_names).
+[`_TokenEmbedding.get_embedding_and_pretrained_file_names`](#mxnet.contrib.text.embedding.TokenEmbedding.get_embedding_and_pretrained_file_names).
```

Comment:
`get_embedding_and_pretrained_file_names` function name too long and tedious IMO.

Reply:
Thanks. Although it is possible to shorten it with some alias, I think this name is clearer.

Reply:
I'm inclined to what @zhreshold says. This is a lot of typing indeed... Consider a more concise name.

Reply:
Since get_embedding_and_pretrained_file_names() takes embedding_name=None as its argument, how about get_pretrained_file_names() ?

Reply:
Sounds good to me

Reply:
Resolved.